sci-mathematics/cgal/metadata.xml

```diff
@@ -1,6 +1,10 @@
 <?xml version='1.0' encoding='UTF-8'?>
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
+  <maintainer type="person">
+    <email>gentoo@aisha.cc</email>
+    <name>Aisha Tammy</name>
+  </maintainer>
```

Comment:
do this in another commit, and add yourself after `sci-mathematics`, and don't forget to add proxy-maint

Reply:
done